pkg/setting/setting.go

```diff
@@ -923,6 +928,18 @@ func (cfg *Cfg) readLDAPConfig() {
 	cfg.LDAPAllowSignup = LDAPAllowSignup
 }
 
+func (cfg *Cfg) readAWSConfig() {
+	awsPluginSec := cfg.Raw.Section("aws")
+	cfg.AWSAssumeRoleEnabled = awsPluginSec.Key("assume_role_enabled").MustBool(false)
+	allowedAuthProviders := awsPluginSec.Key("allowed_auth_providers").String()
+	for _, user := range strings.Split(allowedAuthProviders, ",") {
+		user = strings.TrimSpace(user)
+		if user != "" {
+			cfg.AWSAllowedAuthProviders = append(cfg.AWSAllowedAuthProviders, user)
+		}
+	}
```

Comment:
Total nit, but I find it confusing that the variable is called `user`, when it represents a provider name.

Reply:
good catch again